resume.json

```diff
@@ -1,108 +1,196 @@
 {
   "basics": {
     "name": "Nuvi Pannu",
-    "label": "Programmer",
-    "picture": "",
-    "email": "npannu@gmail.com",
-    "phone": "(912) 555-4321",
-    "website": "http://nuvipannu.com",
-    "summary": "<3",
+    "label": "Developer/Tech ♡er",
+    "picture": "http://i.imgur.com/ns7polk.jpg?1",
```

Comment:
:heart_eyes: Wow, I just noticed and OMG good looking out for making me look less fat haha. :+1:  Amazing eyes :eyes: with great attention to detail :grinning: : You always go above and beyond for me and I am so grateful. You are bestest friend ever!! :purple_heart: